src/gui/admin.py

```diff
@@ -26,6 +24,20 @@ class NewsAdmin(SummernoteModelAdmin):
     summernote_fields = 'text'
 
 
+@admin.action(description='Datei als CSV exportieren')
+def download_csv(modeladmin, request, queryset):
+    import csv
+    f = open('some.csv', 'wb')
+    writer = csv.writer(f)
+    writer.writerow(["code", "country", "ip", "url", "count"])
```

Comment:
I think this goes into the right direction, however the field names need to be adopted to our use cases.